### ISSUE TITLE
streaming not working FF/chrome

### ISSUE THREAD

**Issue by davidschober** [avalonmediasystem/avalon#1361]
Streaming is not working on FF 49.x and chrome 54.x. It seems to be when users are off IU's network. This is true of Spruce and Mallorn. 

to recreate:
* got to https://mallorn.dlib.indiana.edu/media_objects/avalon:3136 OR https://https://spruce.dlib.indiana.edu/media_objects/ab%2F2b%2F23%2Fee%2Fab2b23ee-b3ac-4e3a-8958-da1b5157c416 in chrome or ff while off IU's network
* see that start and end are 0:00 -- 0:00 and won't update or play
* go to safari, (HLS) and note that works
* login to IUs vpn -- per @senpai and note that streaming works. 
* Note Paw Paw works for both on and off networks (less restricted server).

**Comment by mcwhitaker** [comment]
Limitations due to firewall policy prevent remote access to port 1935. We will try to configure red5 on spruce to operate on port 80 on a virtual network interface instead. If this works, we are set. If it does not work, we will sponsor IU accounts to other NU team members (currently only MBK and Chris S have sponsored IU accounts).

**Comment by davidschober** [comment]
@mcwhitaker to sponsor accounts. Created  new ticket for port 80

**Comment by davidschober** [comment]
@mcwhitaker to sponsor IU VPN accounts. created new ticket to track port 80 workaround #1397. Possibly get 1395 open.

**Comment by Senpai** [comment]
Not sure why I'm suddenly involved in this, but I'm backing... away... slowly.

**Comment by mcwhitaker** [comment]
Waiting for personal information from NU users to request creation of accounts.

**Comment by csyversen** [comment]
@mcwhitaker @davidschober ,  @bmquinn is going to be out for a bit, so i think we should probably just wait until karen has access before we can close this issue

**Comment by mcwhitaker** [comment]
@csyversen - I got info from David and Karen and both processes are flowing. 

To simply test on spruce all they need is the IU username and then they can use the VPN. Karen already has hers. I don't know about @davidschober.

But I also would like to set up the developers with the OS-level access on spruce.

**Comment by davidschober** [comment]
@csyverson can you show me how to hook up to IUs VPN? My U/P is fine.

**Comment by carrickr** [comment]
@mcwhitaker brian wheeler controls access to spruce, we can add users but he becomes upset

**Comment by davidschober** [comment]
@mcwhitaker I followed the instructions here https://kb.iu.edu/d/aygt

and all looks good. One note @kdid I had to login to the system and OK my default password first.

**Comment by joncameron** [comment]
@davidschober Based on your last comment can we move this to done?

**Comment by davidschober** [comment]
Yup.
